src/main/java/org/opensearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -179,8 +208,8 @@ ValidationResult<SecurityConfiguration> createOrUpdateAccount(
         try {
             final var username = securityConfiguration.entityName();
             final var content = (ObjectNode) securityConfiguration.requestContent();
-            if (request.hasParam("service")) {
-                content.put("service", request.param("service"));
+            if (request.hasParam("attributes")) {
+                content.put("attributes", request.param("attributes"));
```

Comment:
@peternied  @scrawfor99  with this change to the put request I was able to not use the 'service' in the root of the object. Does that make sense?

Reply:
Looks like this was a bug in the original implemetnation, thanks for finding and fixing.